### PR TITLE
chore(release): initial

### PR DESCRIPTION
**Pull request by amolero-nr**
Initial release.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    - cloud
+    - logging
+    - host
+    - linux
+    - windows
+    - .NET
+    - Java
+    - PHP
+    - NodeJS
+    - apm
```
Comment (rthorn-nr) [inline meta/main.yml:59]: We should remove these galaxy tags for now

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.DS_Store
+.yamllint
+inventory.yml
+playbook.yml
```
Comment (rthorn-nr) [inline .gitignore:4]: Wonder if we need to explicitly exclude these files... we're expecting this role to be installed from galaxy, so it's unlikely that these files will exist in the project root

File: meta/main.yml (hunk 1 above)
Comment (amolero-nr) [reply meta/main.yml:59]: True. Thanks @rthorn-nr!

File: .gitignore (hunk 2 above)
Comment (amolero-nr) [reply .gitignore:4]: Yes, I did that in another commit. Thanks!

**Pull request merged**